demo/app.js

```diff
@@ -18,13 +18,15 @@
 				}
 			}
 		};
-
+		$scope.text="temp";
```

Comment:
Why are these 2 lines for?

Reply:
Sorry, I added those lines for the testing purpose, those lines are not needed.

Reply:
Please update accordingly. Thanks.